relayer/relays/beefy/parameters.go

```diff
@@ -97,19 +99,26 @@ func cleanSignature(input types.BeefySignature) (uint8, [32]byte, [32]byte) {
 }
 
 func (r *Request) generateValidatorAddressProof(validatorIndex int64) ([][32]byte, error) {
-	leaves := make([][]byte, len(r.Validators))
-	for i, rawAddress := range r.Validators {
+	var leaves [][]byte
+	var err error
+	var invalidAddress []string
+	for _, rawAddress := range r.Validators {
 		address, err := rawAddress.IntoEthereumAddress()
 		if err != nil {
-			return nil, fmt.Errorf("convert to ethereum address: %w", err)
+			invalidAddress = append(invalidAddress, util.BytesToHexString(rawAddress[:]))
+			leaves = append(leaves, make([]byte, 0))
```

Comment:
Why do we append empty leaf on error as opposed to leaving it out?

Reply:
The `BeefyEcdsaToEthereum` impl on Polkadot will map the invalid AuthorityId to default(an empty address). More context in https://snowfork.slack.com/archives/G01BGMGLAC9/p1716289299447179?thread_ts=1715552817.012679&cid=G01BGMGLAC9